src/backend/opengl/CMakeLists.txt

```diff
@@ -186,12 +195,18 @@ TARGET_LINK_LIBRARIES(forge
     PRIVATE ${FREETYPE_LIBRARIES}
     PRIVATE ${WTK_LIBRARIES}
     PRIVATE ${OPENGL_gl_LIBRARY}
-    PRIVATE ${GLBINDING_LIBRARIES}
     PRIVATE ${FREEIMAGE_LIBRARY}
     PRIVATE ${X11_LIBS}
     )
 
-ADD_DEPENDENCIES(forge ${glsl_shader_targets} glbinding)
+ADD_DEPENDENCIES(forge ${glsl_shader_targets})
+IF(USE_SYSTEM_GLBINDING)
+    TARGET_LINK_LIBRARIES(forge PRIVATE glbinding::glbinding)
```

Comment:
I believe thats one way CMake is allowing the libraries to be linked. From what I've read, it handles everything, ie. includes, build options (defines, cxx etc), and libraries.
glbinding link: https://github.com/cginternals/glbinding#tips-for-linking